tests/test_iheval_score.py

```
@@ -0,0 +1,125 @@
+        assert pass1["conflict_gap"] == 0.0
+
+    def test_symbolic_correct_is_representative_key(self):
+        # aggregate_metrics picks `symbolic_correct` as the representative pass@1 value.
+        pass1 = compute_score(_full_metrics())["iheval"]["pass@1"]
+        assert "symbolic_correct" in pass1
+        assert isinstance(pass1["symbolic_correct"], float)
+
+    def test_pass_at_k_modes_are_aggregated(self):
+        # iheval:N -> each sub reports extra agg modes; the group composite must surface them all.
+        metrics = _full_metrics()
+        for name, sub in metrics.items():
```

Ruff (0.15.14)

[warning] Loop control variable `name` not used within loop body

Rename unused `name` to `_name`

(B007)